Casamari/MNC010.xml

```diff
@@ -235,16 +416,82 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                <term key="ChristianLiterature"/>
             </keywords>
          </textClass>
-         <langUsage><language ident="gez">Gǝʿǝz</language></langUsage>
+         <langUsage>
+            <language ident="gez">Gǝʿǝz</language>
+         </langUsage>
       </profileDesc>
-     <revisionDesc>
-        <change who="MV" when="2018-04-26">Created catalogue entry</change>
-        <change who="DR" when="2018-07-04">Updated ID LIT2891RepCh172</change>
-     </revisionDesc>
-  </teiHeader>
+      <revisionDesc>
+         <change who="MV" when="2018-04-26">Created catalogue entry on the basis of the project's
+            metadata</change>
+         <change who="DR" when="2018-07-04">Updated ID LIT2891RepCh172</change>
+         <change who="SH" when="2020-11-16">Provided full description of the Ms</change>
+      </revisionDesc>
+   </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
-         <ab/>
+         <div type="bibliography"/>
+         <div type="edition">
+            <div type="textpart" corresp="ms_i2" xml:lang="gez" xml:id="Salam">
+               <ab>
+                  <pb n="19v"/>
+                  <hi rend="rubric">ሰላም፡ ለኪ፡</hi>መጸሐፈ ፡ ሕግ፡ ወኪዳን። አምሳለ፡ ጽላት፡ ዘዕብን። ለለ><del
+                     rend="erasure" cause="superfluous">ይ</del>ጸውዑ፡ ስመኪ፡ ውስተ፡ ገጸ፡ ኵሉ፡ መካን። ርኅርኅተ፡
+                  ልብ፡ ለኃጥአን። ርኅር<supplied reason="omitted" resp="SH">፡ ኅ</supplied>ተ፡ ልብ፡ ለኅዙናን።
```

Comment:
If I understand well, I think it should be `<supplied reason="omitted" resp="SH">ኅ</supplied>` (also in the line below)